api/server.js

```diff
@@ -0,0 +1,130 @@
+const express = require('express');
+const userDB = require("../data/helpers/userDb.js");
+const postDB = require("../data/helpers/postDb");
+const server = express();
+
+server.use(express.json());
+
+function uppercaser(req, res, next) {
```

Comment:
This is a very efficient middleware as it is only three lines.  Most functions I've seen so far are much longer.  Great job completing this middle ware with such few lines.